app/components/pages/Home.jsx

```diff
@@ -25,10 +26,12 @@ class Home extends React.Component {
   }
 
   render() {
+    const { t } = this.props;
+
     return (
       <div className="posts">
         <section className="post">
-          <h1 className="post-title">Programa de Mentorados de Engenharia Informática - FEUP</h1>
+          <h1 className="post-title">{t("home:title")}</h1>
```

Comment:
Why is there a `home:` prefix here?

Reply:
Holdover from when I was trying stuff out. Not necessary as we import the `home` namespace in the `translate([ "home" ])` HOC. However, if, we had imported multiple namespaces and not defined a default, we'd need to access translation strings like this. Removing namespace.